addons/arsenal/ACE_Arsenal_Sorts.hpp

```diff
@@ -0,0 +1,64 @@
+class GVAR(sorts) {
```

Comment:
Is the `scope` entry and as a result the `sortBase` class necessary? I think its reasonable to have all sorting algorithms defined here be public.

Also, fix spaces after commas in this file.

Reply:
If we want to add properties in the future this class would be useful, it also keeps it inline with the rest of `arsenal`